src/ecmult_gen_impl.h

```diff
@@ -49,39 +49,39 @@ static void secp256k1_ecmult_gen_context_build(secp256k1_ecmult_gen_context *ctx
 
     /* compute prec. */
     {
-        secp256k1_gej precj[1024]; /* Jacobian versions of prec. */
+        secp256k1_gej precj[ECMULT_GEN_PREC_N * ECMULT_GEN_PREC_G]; /* Jacobian versions of prec. */
         secp256k1_gej gbase;
         secp256k1_gej numsbase;
         gbase = gj; /* 16^j * G */
```

Comment:
there are some comments that should be changed tere and up to line 67

Reply:
I updated the comments, assuming you meant to change, for example, `16` to `PREC_G`. Let me know if there is something in addition.